### PR TITLE
view admin

### PR DESCRIPTION
**Pull request by FT-Mystogan**
## Related Tickets:
- [#46284](https://edu-redmine.sun-asterisk.vn/projects/hn_oe49-intern-course-management/issues/46284)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by FT-Mystogan** [comment]
ready

**Comment by FT-Mystogan** [comment]
ready

File: resources/views/layouts/admin.blade.php [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    <!-- BEGIN: load jquery -->
+    <script src="{{ asset('bower_components/bower_project1/js/admin/jquery.js') }}" type="text/javascript"></script>
+    <!-- END: load jquery -->
+    <script src="{{ asset('bower_components/bower_project1/js/admin/app.js') }}" type="text/javascript"></script>
+    <script type="text/javascript">
+        $(document).ready(function() {
+            setupLeftMenu();
+            setSidebarHeight();
+        });
+    </script>
```
Comment (manhtd-1083) [inline resources/views/layouts/admin.blade.php:19]: bỏ script vào file js riêng đi bn

File: resources/views/admin/student/show.blade.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+            </div>
+        </div>
+    </div>
+    <script type="text/javascript">
+        $(document).ready(function() {
+            setupLeftMenu();
+            $('.datatable').dataTable();
+            setSidebarHeight();
+        });
+    </script>
```
Comment (manhtd-1083) [inline resources/views/admin/student/show.blade.php:39]: bỏ script vào file js riêng

**Comment by FT-Mystogan** [comment]
ready

File: resources/views/admin/course/add.blade.php [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+                    <table class="form">
+                        <tr>
+                            <td>
+                                <label>{{ __('name') }}</label>
+                            </td>
+                            <td>
+                                <input type="text" placeholder="{{ __('Enter Course name...') }}" class="medium"
+                                    name="name" id="name" />
+                                @error('name')
+                                    <span style="color: red">{{ $message }}</span>
```
Comment (manhtd-1083) [inline resources/views/admin/course/add.blade.php:24]: bỏ style ra file css riêng nha.

**Comment by FT-Mystogan** [comment]
ready

**Review by huyenttm-0901 — APPROVED** [review]

**Pull request merged**